mods/d2k/rules/aircraft.yaml

```diff
@@ -51,6 +51,9 @@ carryall:
 	Buildable:
 		Queue: Aircraft
 		BuildPaletteOrder: 120
+	RevealOnDeath:
```

Comment:
Sorry, my mistake.  I forgot that carryalls don't have any vision of their own, so revealing on their death looks weird.  Can you drop this again?